packages/instant/src/style/theme.ts

```diff
@@ -12,6 +12,8 @@ export enum ColorOption {
     feintGrey = 'feintGrey',
     darkGrey = 'darkGrey',
     white = 'white',
+    lightOrange = 'lightOrange',
```

Comment:
Per convo, at some point we should separate out the colors which will be customizable by the widget versus the colors which are not.  I think at this point the only customizable color is `primaryColor`

Reply:
Yea I ran into some color issues in my branch too. Seems there are different requirements now. I would rather we tackle this when my other branch (https://github.com/0xProject/0x-monorepo/pull/1131/files) is merged because it adds the concept of getting color from `assetMetaData` and that needs to fit into whatever scheme we come up with for dealing with changing colors.